### ISSUE TITLE
May I use a new dataset as the test or validation set instead of splitting it from the training set?

### ISSUE THREAD

**Issue by lili084** [DataCanvasIO/HyperGBM#105]
May I use a new dataset as the test or validation set instead of splitting it from the training set? In other words, can I decide on my own how to load the training and test sets? Thank you very much for your answer.

**Comment by lixfz** [comment]
Yes, you can. see https://hypergbm.readthedocs.io/en/latest/example.html as startpoint
